src/services/EdFi.DataManagementService.Api/Core/Exceptions/BaseDetailedException.cs

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.DataManagementService.Api.Core.Exceptions
```

Comment:
please use newer namespace format. In Visual Studio, you can add `;` at the end of line 6, hit enter, and it will automatically fixup the code below it.